src/components/Nav/index.tsx

```diff
@@ -92,16 +94,37 @@ const Nav: FC<IProps> = ({ path }) => {
                   variant="ghost"
                   isSecondary
                   px={1.5}
+                  _hover={{
+                    transform: "rotate(30deg)", // Rotate the icon to 30 degrees on hover
```

Comment:
Re-reading the conversations from the issue, I think we ended up agreeing on just [rotating this 10 degrees](https://github.com/ethereum/ethereum-org-website/pull/10811#issuecomment-1665601093).
```suggestion
                    transform: "rotate(10deg)",
```